R/bump-version.R

```diff
@@ -95,17 +95,17 @@ get_main_branch_remote <- function(remote, repo) {
   basename(as.character(remotes$symref[remotes$ref == "HEAD"]))
 }
 
-get_main_branch_config <- function(repo) {
+  get_main_branch_config <- function(repo) {
+  #retrieve config of repo, filter down to init.defaultbranch values
   config <- gert::git_config(repo)
   init <- config[config$name == "init.defaultbranch", ]
-  local <- init[init$level == "local", ]
 
-  if (length(local)) {
-    return(local$value)
+  #return local default branch if it exists, otherwise default to global
```

Comment:
I think the comments can be removed as the code is clear enough: https://blog.r-hub.io/2023/01/26/code-comments-self-explaining-code/

Reply:
I think you're right. What would be more helpful is a comment explaining the purpose behind this internal function. It's somewhat redundant at the moment but if you support more configurations in the future it will come in handy.

Reply:
thank you! Can you remove the comments then? An alternative to adding a comment about the function would be to improve its name.

Reply:
Okay I think I got it right this time. Github is almost as complicated as git itself.

Reply:
For sure, there's quite the learning curve. Some resources although you didn't ask :innocent: 

- https://happygitwithr.com/
- https://usethis.r-lib.org/articles/pr-functions.html
- https://masalmon.eu/tags/git/